sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CSharpSyntaxHelper.cs

```diff
@@ -247,6 +252,11 @@ public static bool IsStringEmpty(this ExpressionSyntax expression, SemanticModel
             }
         }
 
+        public static bool IsMethodInvocation(this InvocationExpressionSyntax expression, KnownType type, string methodName, SemanticModel semanticModel) =>
+            semanticModel.GetSymbolInfo(expression).Symbol is IMethodSymbol methodSymbol &&
+            methodSymbol.IsInType(type) &&
+            methodName.Contains(methodSymbol.Name);

```

Comment:
Why is this `methodName.Contains(methodSymbol.Name)` instead of testing for equality?